openpype/plugins/publish/extract_burnin.py

```diff
@@ -78,6 +78,12 @@ class ExtractBurnin(publish.Extractor):
     options = None
 
     def process(self, instance):
+        # Instance should be integrated on a farm
+        if instance.data.get("farm"):
+            self.log.info(
+                "Instance is marked to be processed on farm. Skipping")
+            return
```

Comment:
Note that the logs in the artist facing report do NOT show what plugin is a log is related to. So with all these skipping they'd just see e.g. five lines saying:
> Instance is marked to be processed on farm. Skipping
> Instance is marked to be processed on farm. Skipping
> Instance is marked to be processed on farm. Skipping
> Instance is marked to be processed on farm. Skipping

Having no idea of what actually got skipped.
This is either a design issue with the new publisher OR we should log what is actually being skipped, like "Skipping burnin extraction", "Skipping colorspace transcoding", etc.